### ISSUE TITLE
Improvement ideas to select an area

### ISSUE THREAD

**Issue by SFX72** [Mahlet-Inc/hobbits#103]
Hi,
just an idea to select an area in a view with the mouse ; 

first onclick (left) define the start of the area in the view
SHIFT + onclick (left) define the end of the area

After use the 'Extractor' (Operator plugin) to choose 'Extract Section" or other extract operations. 
or may be new 'right click' access for extract operations.

would this be technically feasible?

Best regards

**Comment by hello-adam** [comment]
Yeah, there is a similar functionality in some views where you can right click and then highlight some contiguous bits. Adding extracting to the right click context menu could probably work in a similar way.

I do agree that some kind of intuitive and fast left click highlighting would be good in many of the views. It's all definitely technically feasible, but I might need to sit down with this issue one day and really think about the best way to do it so that it is actually intuitive and fast.
